src/main/java/org/example/UtilityClass.java

```diff
@@ -0,0 +1,31 @@
+package org.example;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+
+public class UtilityClass {
```

Comment:
let's follow single responsibility principle: divide this class to CommonUtils and WebUtils